### DISCUSSION TITLE
Why does the trade priod starts when the deposit tx is published?

### DISCUSSION THREAD

**Discussion by imfiesh** [bisq-network/bisq#6437] [General]
Why is the trade period counted from when the deposit tx is published rather than when it's confirmed? For instant trades this is a problem. Its not at all unusual for it to take >30 minutes for the tx to be confirmed, so at that point the trader only has <30 minutes left to transfer altcoin / confirm receipt of altcoin. For me personally it has happened at least twice that it has taken _more_ than 60 minutes, with the result that once Bisq lets the trader transfer/confirm, it's already too late, with a scary red warning notice being shown. At least it's not good UX.

**Comment by pazza83** [comment]
I think it was also reported here: https://bisq.community/t/instant-altcoin-trades-trading-window-and-remaining-time/11682/2

I think @jmacxx might have looked into it?

**Reply by ghost** [reply]
Yes, came across the bug recently, a [fix](https://github.com/bisq-network/bisq/pull/6422) is going into the next software release.
